### PR TITLE
Make the required changes for relay support

### PR DESCRIPTION
**Pull request by robertdolca**
Now you can configure AWS SES as relay for postfix.

### CODE REVIEW

File: start-mailserver.sh [hunk 1]
```diff
@@ -173,6 +173,26 @@ case $DMS_SSL in
 
 esac
 
+if [ -f /tmp/postfix/main.cf ]; then
+  while read line; do
+    postconf -e "$line"
```
Comment (tomav) [inline start-mailserver.sh]: Could you add an integration test on this, like overriding a default value.
Add a `test/main.cf` with one override. Do not forget to update `make clean` task in the `Makefile`.

File: start-mailserver.sh [hunk 2]
```diff
@@ -173,6 +173,26 @@ case $DMS_SSL in
...
+  done < /tmp/postfix/main.cf
+  echo "Loaded '/tmp/postfix/main.cf'"
+else
+  echo "==> Warning: '/tmp/postfix/main.cf' is not provided. No extra postfix settings loaded."
```
Comment (tomav) [inline start-mailserver.sh]: This is more an `Info` than `Warning` since these settings are optional.

File: start-mailserver.sh [hunk 3]
```diff
@@ -173,6 +173,26 @@ case $DMS_SSL in
...
+fi
+
+if [ ! -z "$SASL_PASSWD" ]; then
```
Comment (tomav) [inline start-mailserver.sh]: How can we add a test on this part to be sure that it won't be broken?

**Comment by tomav** [comment]
Thank for this @robertdolca! :+1: 
Let me know if I can help regarding comments left above.

**Comment by tomav** [comment]
@robertdolca how are you? Did you have time to work on comments above?
Let me know if I can help. 
Thank you.

**Comment by robertdolca** [comment]
Your comment seem justified. I didn't have time to implement the changes yet.
Thanks @tomav

**Comment by tomav** [comment]
@robertdolca still alive? Could you let us know what's going on. Thank you.

**Comment by robertdolca** [comment]
Just a very busy schedule. If you have time and you are wolling to you can make the proposed changes and I will take a look. If not al will make them asap.

**Comment by tomav** [comment]
If I have time this week end, I'll let you know here before to work on it.

**Pull request merged**